src/app/server/index.js

```diff
@@ -1,16 +1,12 @@
-/* global __DIST__ */
+/* global USE_DEFAULT_KOA_MIDDLEWARES HAS_KOA_MIDDLEWARES KOA_MIDDLEWARES  */
```

Comment:
I think (in general) we should add a comment about these and how they are used by DefinePlugin